docs/docs/index.md

```diff
@@ -57,7 +57,7 @@ var myChart = new Chart(ctx, {
 
 ## Contributing
 
-Before submitting an issue or a pull request to the project, please take a moment to look over the [contributing guidelines](./developers/contributing.md) first.
+Before submitting an issue or a pull request to the project, please take a moment to look over the [contributing guidelines](./docs/docs/developers/contributing.md) first.
```

Comment:
does not work
![image](https://user-images.githubusercontent.com/39033624/101243287-61c82880-36ff-11eb-9035-5aeb691b88b5.png)

Reply:
will make changes